src/Context.re

```diff
@@ -0,0 +1,25 @@
+type t = {
+  autoClosingPairs: AutoClosingPairs.t,
+  bufferId: int,
+  width: int,
+  height: int,
+  leftColumn: int,
+  topLine: int,
+  cursors: list(Cursor.t),
+  lineComment: option(string),
+  tabSize: int,
+  insertSpaces: bool,
+};
+
+let default = () => {
+  autoClosingPairs: AutoClosingPairs.empty,
+  bufferId: Buffer.getCurrent() |> Buffer.getId,
+  width: Window.getWidth(),
+  height: Window.getHeight(),
+  leftColumn: Window.getLeftColumn(),
+  topLine: Window.getTopLine(),
+  cursors: [Cursor.get()],
+  tabSize: Options.getTabSize(),
+  insertSpaces: Options.getInsertSpaces(),
+  lineComment: None,
+};
```

Comment:
Is this really a "default"? It looks more like the current state to me, and whether or not it should be used as a default is a decision for the consumer.

Reply:
Good point, `current` would be more descriptive 👍